src/alcotest-engine/alcotest_engine.mli

```diff
@@ -51,3 +51,6 @@ module Private : sig
   module Utils = Utils
   module Pp = Pp
 end
+
+module Higher = Higher
```

Comment:
Do we really need to use `Higher`? This brings a new level of complexity which might make future contributions and maintenance a bit more difficult that it is already.

Reply:
We don't. I like that it avoids the need for `Platform.MAKER` in favour of `Platform.S`, but there's not a huge benefit.